### PR TITLE
Leaveslip Issues

### PR DESCRIPTION
**Pull request by davidhtien**
* change color of trainee who submitted
* add delete functionality to ta detail view for leaveslips
* add back period select for leaveslip forms, instead deselect all events when switching periods (this includes the roll form but I don't think this is an issue? @JoannaRumbley @finben )
* make default sort order for ta list view for leaveslips ascending (right now it's descending)

### CODE REVIEW

**Comment by davidhtien** [comment]
This might be a little excessive? Someone checkout it and let me know what they think

File: ap/leaveslips/templates/leaveslips/group_update.html [hunk 1]
```diff
@@ -4,10 +4,31 @@
...
+  {{ form.media }}
+{% endblock %}
+
 {% block leaveslip_script %}
   <script>
+    var submitter_visible = {{ submitter_in_slip|yesno:"true,false" }};
     $(document).ready(() => {
-      $('#id_trainees').select2({width: '100%'});
+      $("label[for|='id_trainees']").append(
+        '<span id="submitter" style="display: none; margin-left: 10px; font-size: 12px;">Submitter: Shaun Pan</span>');
```
Comment (iamsitting) [inline ap/leaveslips/templates/leaveslips/group_update.html]: Does Shaun Pan need to be hard-coded? 😄

Comment (davidhtien) [reply ap/leaveslips/templates/leaveslips/group_update.html]: LOL my b

**Comment by iamsitting** [comment]
@davidhtien let me know when this is finished.

**Comment by JoannaRumbley** [comment]
What is the "roll form"? This change is just for leave slips. Roll form should probably remain how it was before the period select drop-down was taken out...

File: libraries/react/scripts/actions.js [hunk 2]
```diff
@@ -21,6 +21,7 @@ export const selectPeriod = (period) => {
   return (dispatch, getState) => {
     let dateDetails = getDateDetails(getState())
     dispatch(changeDate((period - dateDetails.period) * 14))
+    dispatch(deselectAllEvents())
```
Comment (davidhtien) [inline libraries/react/scripts/actions.js]: @JoannaRumbley @finben 
This is what I'm talking about, whenever the period selector is used we will deselect all events, this of course includes the roll view in addition to the leaveslip views for personal attendance

**Pull request merged**